cmd/extensions/main.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Controller for gameservers
+// Extensions for gameservers
```

Comment:
```suggestion
// Extensions for the Agones system
```
Nit: Original comment was probably debt from when I initially wrote the controller.

Reply:
Done.